src/qt/bitcoingui.cpp

```diff
@@ -701,6 +707,20 @@ void BitcoinGUI::openClicked()
     }
 }
 
+void BitcoinGUI::openWalletClicked()
+{
+    QString fileName = QFileDialog::getExistingDirectory(this, tr("Open Wallet"),
```

Comment:
Use `QDir::toNativeSeparators` to convert `/` to `\` on Windows

Reply:
Done.

Reply:
^^